calendar/calendar-impl/impl/src/java/org/sakaiproject/calendar/impl/BaseCalendarService.java

```diff
@@ -1587,6 +1587,9 @@ public String merge(String siteId, Element root, String archivePath, String from
 									String newId = getUniqueId();
 									element3.setAttribute("id", newId);
 
+									// Do not import events with hard-coded links that will break
```

Comment:
You don't really need this comment. shouldMergeEvent will maybe do more than this, so the comment may well soon be out of date.